app/models/reports/facility_state.rb

```diff
@@ -24,5 +24,9 @@ def period
     scope :with_patients, -> {
       where("cumulative_registrations > 0 OR cumulative_assigned_patients > 0 OR monthly_follow_ups > 0")
     }
+
+    scope :with_htn_or_diabetes_patients, -> {
```

Comment:
This scope is used to get active facilities. 

Once we rename the htn fields in the materialized views, this scope should not be necessary and we can revert to the original form.

Reply:
As in we would have `cumulative_registrations`, `cumulative_htn_registrations` and `cumulative_diabetes_registrations`?

I was wondering if DM reports would look at `cumulative_diabetes_registrations` instead of `cumulative_registrations` in that case, since HTN reports will look at `cumulative_htn_registrations`.

Reply:
We will be appending `htn` to all the fields related to hypertension and everything else will be the same